app/src/pages/inside/common/infoLine/infoLine.jsx

```diff
@@ -6,6 +6,7 @@ import styles from './infoLine.scss';
 import { BarChart } from './barChart';
 import { Duration } from './duration';
 import { DefectTypeBlock } from './defectTypeBlock';
+import { getDuration } from '../../../../common/utils/timeDateUtils';
```

Comment:
It is better to use absolute import here:
`import { getDuration } from 'common/utils/timeDateUtils';`